images/win/scripts/Installers/Install-PostgreSQL.ps1

```diff
@@ -0,0 +1,59 @@
+function Install-PostgreSQL {
+    $installPath= choco install postgresql | Select-String "Installed to"
+    return $installPath.Line.split("'")[1]
+}
+
+function Start-PostgreSQL {
+    param(
+        [String]$PostgresPath
+    )
+    $pgbin=Join-path $PostgresPath "bin"
+    $pgdata=Join-path $PostgresPath "data"
+    $startPostgres=Join-path $pgbin "pg_ctl.exe"
+    Start-Process -FilePath $startPostgres -ArgumentList ("-D", "$pgdata", "start")
+    Write-Host "PostgreSQL has been successfully started."
+}
+
+function Ready-PostgreSQL {
+    param(
+        [String]$PostgresPath
+    )
+    $pgbin=Join-path $PostgresPath "bin"
+    $pgisreadypath=Join-path $pgbin "pg_isready.exe"
+    $pgready = Start-Process -FilePath $pgisreadypath -Wait -PassThru
+    $exitCode = $pgready.ExitCode
+        if ($exitCode -eq 0)
+        {
+          Write-Host -Object "PostgreSQL has been successfully installed."
+        }
+        else
+        {
+          Write-Host -Object "PostgreSQL is not ready. Exitcode: $exitCode"
+          exit $exitCode
+        }
+}
+
+function Version-PostgreSQL {
+    param(
+        [String]$PostgresPath
+    )
+    $pgbin=Join-path $PostgresPath "bin"
+    $pgconfig=Join-path $pgbin "pg_config.exe"
+    $psqlversion=Start-Process -FilePath $pgconfig -ArgumentList ("--version")-Wait -PassThru
+    return $psqlversion
+}
+
+$psqlDir=Install-PostgreSQL
+$psqlVersion=Version-PostgreSQL -PostgresPath $psqlDir
+Start-PostgreSQL -PostgresPath $psqlDir
```

Comment:
Do we really want to start it? I wonder because previously we had the same situation with mySQL and we have to disable it by default

Reply:
I have added stop action instead of start.